contracts/container/contract.go

```diff
@@ -195,6 +207,12 @@ func Update(nefFile, manifest []byte, data any) {
 // must contain information about an object placed to a container that was
 // created using [Put] ([PutMeta]) with enabled meta-on-chain option.
 func SubmitObjectPut(metaInformation []byte, sigs [][]interop.Signature) {
+	ctx := storage.GetContext()
+	proxyH := storage.Get(ctx, proxyContractKey).(interop.Hash160)
```

Comment:
...unlike this one, which is sequential and the cost is _much_ higher here. If we care about breaking the circular dependency I'd leave this getter here and go with NNS for `verify()`.

Reply:
> we care about breaking the circular dependency

Currently it is solved, i do not like how, but it works. i would put NNS everywhere as a simplier solution, but this PR is about optimization so storage getter everywhere now. not sure i understand what current state of this thread? leaving getters?